.gitignore

```diff
@@ -35,6 +35,9 @@ nosetests.xml
 .project
 .pydevproject
 
+# PyCharm
+.idea
```

Comment:
Thanks, I will run checks on Read the Docs today, I didn't know we could run it on branches. For `.idea` my understanding is that without the trailing slash it will match files, directories, symlinks, etc. by that name. I figured we didn't want any of those things if they should exist, but if there's a reason why we might want to let one of those through I'd be happy to change it.